Makefile

```diff
@@ -1,33 +1,36 @@
+DART_DEV=https://dev.dart.harvard.edu
+DART_STAGE=https://stage.dart.harvard.edu
```

Comment:
I'm not really sure why the `STAGE_URL` was used in some of the make paths below... could we just use the same URL and invoke the makefile to use that URL? That can be reconfigured later, I suppose, if we want different defaults for different make paths, but we're a good bit of work away from defaulting to a stage environment still.

Reply:
I didn't spend any time exploring why RG had two different URLs, but figured that the low-hanging-fruit was just to add it. I can definitely revisit this in DART-106 when I put a solid effort into understanding RG's different Make recipes.